docs/src/mdx-components.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {
+  Heading,
+  Text,
+  TextLink,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+} from "@kiwicom/orbit-components";
+import { InlineCode, Pre } from "./components/Code";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const p = Text;
+export const h1 = (props: Props) => <Heading as="h1" type="display" {...props} />;
```

Comment:
I don't think I'd support h1s in Markdown. I'd enforce titles in frontmatter and lint out any h1s. That makes it easier to use the data for other purposes.

Reply:
I agree, I'll remove this mapping.